### PR TITLE
feat: use FLSS for live loan image filter searches when supported MP-732

### PR DESCRIPTION
**Pull request by emuvente**
Use FLSS for all filter searches that it supports, with a fallback to the legacy loan search if using a sort option that only works there.

### CODE REVIEW

File: server/util/live-loan/live-loan-fetch.js [hunk 1]
```diff
@@ -197,12 +233,47 @@ const getFilterArrays = filterString => {
...
+	const option = options.find(o => o?.name?.toLowerCase() === value);
+	if (!option) {
+		warn(`Unknown ${name} "${value}"`);
+	}
+	return option;
+};
+
+// Takes a string like: "sort_expiringSoon,gender_female"
+// and extracts the sort option, e.g. "expiringSoon"
+async function parseSortString(sortString, sortOptions) {
```
Comment (dyersituations) [inline server/util/live-loan/live-loan-fetch.js:247]: Does this method need `async`?

Comment (emuvente) [reply server/util/live-loan/live-loan-fetch.js:247]: No, looks like it does not

Comment (emuvente) [reply server/util/live-loan/live-loan-fetch.js]: https://github.com/kiva/ui/pull/5479/commits/662bb2adc063e9ca7268ccda98ac6bc94bcc9724

**Review by dyersituations — APPROVED** [review]
Approved pending passing tests.

**Review by roger-in-kiva — APPROVED** [review]

**Review by mcstover — APPROVED** [review]

**Pull request merged**